apps/full/e2e/runner.ts

```diff
@@ -1,9 +1,181 @@
+const TASK_TARBALL = `${RUNFILES}/_main/services/task/src/main/tarball/tarball.tar`;
+const TASK_TAG = "lab/task:latest";
+const JOURNAL_TARBALL = `${RUNFILES}/_main/services/journal/src/main/tarball/tarball.tar`;
+const JOURNAL_TAG = "lab/journal:latest";
+const AUTOJOURNAL_TARBALL = `${RUNFILES}/_main/services/autojournal/src/main/tarball/tarball.tar`;
+const AUTOJOURNAL_TAG = "lab/autojournal:latest";
+const PROXY_TARBALL = `${RUNFILES}/_main/infra/local/proxy/tarball/tarball.tar`;
+const PROXY_TAG = "lab/proxy:latest";
+
+const DOCKER = new Dockerode();
 
 async function main() {
+  const network = await new Network().start();
+
+  await new KafkaContainer()
+    .withNetworkAliases("broker")
+    .withNetwork(network)
+    .start();
+
+  await new GenericContainer("confluentinc/cp-schema-registry:7.5.3")
+    .withNetwork(network)
+    .withNetworkAliases("schema-registry")
+    .withEnvironment({
+      SCHEMA_REGISTRY_HOST_NAME: "schema-registry",
+      SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS: "broker:9092",
+    })
+    .start();
+
+  const taskContainer = loadContainer(TASK_TARBALL, TASK_TAG).then(
+    (container) =>
+      container
+        .withNetwork(network)
+        .withExposedPorts({
+          host: 3001,
+          container: 3001,
+        })
+        .withEnvironment({
+          SPRING_KAFKA_BOOTSTRAP_SERVERS: "broker:9092",
+          SPRING_KAFKA_PROPERTIES_SCHEMA_REGISTRY_URL:
+            "http://schema-registry:8081",
+        })
+        .withWaitStrategy(Wait.forLogMessage("Started TaskApplication"))
+        .start(),
+  );
+
+  const journalContainer = loadContainer(JOURNAL_TARBALL, JOURNAL_TAG).then(
+    (container) =>
+      container
+        .withExposedPorts({
+          host: 3003,
+          container: 3003,
+        })
+        .withWaitStrategy(Wait.forLogMessage("Started JournalApplication"))
+        .start(),
+  );
+
+  const autojournalContainer = loadContainer(
+    AUTOJOURNAL_TARBALL,
+    AUTOJOURNAL_TAG,
+  ).then((container) =>
+    container
+      .withNetwork(network)
+      .withExposedPorts({
+        host: 3002,
+        container: 3002,
+      })
+      .withEnvironment({
+        LAB_SERVICES_TASK_GRAPHQL_URL:
+          "http://host.docker.internal:3001/graphql",
+        LAB_SERVICES_JOURNAL_GRAPHQL_URL:
+          "http://host.docker.internal:3003/graphql",
+        SPRING_KAFKA_BOOTSTRAP_SERVERS: "broker:9092",
+        SPRING_KAFKA_PROPERTIES_SCHEMA_REGISTRY_URL:
+          "http://schema-registry:8081",
+      })
+      .withExtraHosts([
+        { host: "host.docker.internal", ipAddress: "host-gateway" },
+      ])
+      .withWaitStrategy(Wait.forLogMessage("Started AutoJournalApplication"))
+      .start(),
+  );
+
+  const schema = fs.readFileSync(
+    `${RUNFILES}/_main/infra/local/router/schema/supergraph.graphql`,
+  );
+  const routerConfig = fs.readFileSync(
+    `${RUNFILES}/_main/infra/local/router/config/router.conf`,
+  );
```

Comment:
Hopefully this can be avoided if `withCopyFilesToContainer` can be configured to follow symlinks instead of archiving and transferring them as is.

Tracking issue: https://github.com/testcontainers/testcontainers-node/issues/713